python-threatexchange/threatexchange/cli/hash_cmd.py

```diff
@@ -130,7 +166,15 @@ def execute(self, settings: CLISettings) -> None:
 
         for file in self.files:
             with open(file, "rb") as f:
-                image_bytes = f.read()
+                if (
+                    self.content_type.get_name() == "photo"
+                    and self.preprocess == "unletterbox"
```

Comment:
nit: Instead, in the `__init__` validate that unletterbox can only be used with photo, which will save you many future checks like this.